RunningOrder/Sprint/Models/Sprint.swift

```diff
@@ -23,3 +23,40 @@ extension Sprint {
 
 extension Sprint: Equatable { }
 extension Sprint: Hashable { }
+
+struct SearchSection: Identifiable {
+    var id: UUID { return UUID() }
```

Comment:
```suggestion
    var id: UUID = UUID()
```

why a computed ? don't you want id to be fixed ?
If this is only for conforming to Identifiable, the goal is to have SearchSection unique : can two sections have the same name in the view ? I don't think so consider having
`var id: String { name }` in an extension for conformance